src/rolodex/rolodexPrinter.js

```diff
@@ -1,11 +1,23 @@
 import people from './people.json';
 
+const{name, email, phone} = person;
```

Comment:
`person` is out of scope here, so this will cause an error because the variable is not defined.